### PR TITLE
Squwak Assignments and little updates

### PR DESCRIPTION
**Pull request by pa-ulex**
Hello,

I just loaded up the FIS Package and noticed a few things and they are or will be fixed,

- CCAMS was outdated, updated in this Pull Request
- SquwakIndicator was outdated, updated in this Pull Request and also added FSW Squwak
- FSW uses 7745 IRL looking at FlightRadar

Optional Things that can be done

- [ ] Adding IDO (IDN) and ISW (IDW) to login list

### CODE REVIEW

**Comment by hannesaltmann** [comment]
@pa-ulex does this change to the sector naming reflect recent real-world changes? 

As far as I know, the sectors are named as follows:

- ISW = Sector Hamburg Overload (former Seewelle)
- IDO = Sector München Overload

The sector indicators are not connected to a major aerodrome in those two cases.

**Comment by pa-ulex** [comment]
> As far as I know, the sectors are named as follows:
> 
> * ISW = Sector Hamburg Overload (former Seewelle)
> * IDO = Sector München Overload
> 
> The sector indicators are not connected to a major aerodrome in those two cases.

@hannesaltmann No, I just changed them, because for me it would only make sense to name them like that. I didn’t know that they were overload sectors, because in real life the day I looked the squwaks were used

**Comment by hannesaltmann** [comment]
The Squawks in this PR seem to be correct. I just would stick to the real-world sector names.

**Comment by pa-ulex** [comment]
Also with the update the ese and sct need to be updated. Tegel is still in there.

File: EDXX/Plugins/SquawkIndicator/squawks.json
Comment (hannesaltmann) [inline EDXX/Plugins/SquawkIndicator/squawks.json:1]: Wrong sector names here

**Review by hannesaltmann — CHANGES_REQUESTED** [review]

**Review by hannesaltmann — APPROVED** [review]

**Review by hannesaltmann — APPROVED** [review]

**Comment by pa-ulex** [comment]
Is it possible if someone could review the PR and merge it into the branch or shall it be in another one for 2408?

**Pull request merged**